great_expectations/rule_based_profiler/helpers/configuration_reconciliation.py

```diff
@@ -0,0 +1,77 @@
+import logging
```

Comment:
Moved to prevent circular import, since Variables are reconciled at both the `Profiler` and `Rule` level. Brought into a seaparate module they can both access.